contrib/macdeploy/macdeployqtplus

```diff
@@ -586,7 +586,12 @@ if len(config.fancy) == 1:
         sys.exit(1)
     
     try:
-        fancy = plistlib.readPlist(p)
+        # the old plistlib API was removed in python3.9, use the new API instead
+        if (sys.version_info.major, sys.version_info.minor) >= (3, 9):
```

Comment:
no need for this check. This has been added in python 3.4 https://docs.python.org/3.4/library/plistlib.html#plistlib.load

And we don't even support 3.4 anymore.

Reply:
You'r right. 3.5 is our minimum anyways.
Removed the check.